Userland/Utilities/pmemdump.cpp

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2021, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/String.h>
+#include <AK/StringUtils.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+#include <assert.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+static bool try_set_offset_and_length_parameters(String const& arg_offset, String const& arg_length, u64& offset, u64& length)
+{
+    // TODO: Add support for hex values
+    auto possible_offset = arg_offset.to_uint<u64>();
+    auto possible_length = arg_length.to_uint<u64>();
+    if (possible_offset.has_value() || possible_length.has_value()) {
```

Comment:
shouldn't this be an `&&`? in which case, this could be more neatly written in the 'early return' style